src/context.js

```diff
@@ -29,16 +54,35 @@ type ConsumerProps = {
   children: Function
 };
 
-class LoadsConsumer extends React.Component<ConsumerProps> {
+class LoadsStateConsumer extends React.Component<ConsumerProps> {
   render = () => {
     const { cacheKey, children } = this.props;
+
     return (
       <Consumer>
         {context => {
           return children({
             ...context.data[cacheKey],
             hasResponseInCache: typeof context.data[cacheKey] !== 'undefined',
-            setResponse: data => context.setResponse(cacheKey, data)
+            setResponse: data => context.setStaticResponse(cacheKey, data)
+          });
+        }}
+      </Consumer>
+    );
+  };
+}
+class LoadsLocalStorageConsumer extends React.Component<ConsumerProps> {
```

Comment:
Again, any advantage for having two separate consumers? My thinking was that it would first check if the `cacheKey` exists in localStorage, if it doesn't, just use context state.